### DISCUSSION TITLE
Need Mod Ver**😴

### DISCUSSION THREAD

**Discussion by Kenshinz0** [Chokage/Fear-and-Hunger-2-Termina-Mobile#3] [General]
Thank's a lot for release Mobile Version, but i need to enjoy the game, i need to kill all boss without lost my arm🥹
Just drop the link if u need coffee.
@Chokage 
Sorry for bad english

**Comment by setzer333** [comment]
Hablas de una versión con trucos como armas y otras cosas? Podrías decirme que artículo o equipamiento evita el corte , como la salmonsnake soul en F&H1? Talvez pueda ayudarte 

Google translate:
Are you talking about a version with cheats like weapons and other things? Could you tell me what item or equipment prevents the cut, like the salmonsnake soul in F&H1?
